src/include/daos_srv/vos.h

```diff
@@ -433,15 +433,20 @@ vos_aggregate(daos_handle_t coh, daos_epoch_range_t *epr,
  * the caller.
  *
  * \param coh		[IN]	Container open handle
- * \param epr		[IN]	The epoch range to discard
- *				keys to discard
+ * \param oidp		[IN]	Optional oid for oid specific discard.  Aggregation should be
+ *				disabled before calling this function in this mode.  This simply
```

Comment:
How can we guarantee the VOS aggregation is disabled if it is not in test?

Reply:
In subsequent patch, @wangdi1 will need to ensure that it is disabled on the reintegrated target from before the discard phase until after reintegration is complete

Reply:
Then can we check vos_container::vc_in_aggregation inside vos_discard()? that will be more safe?

Reply:
I could assert that we are not in aggregation if you prefer?   This code is presently unused though so I could also do it in a subsequent patch?